src/index.js

```diff
@@ -53,7 +54,13 @@ function prettify(text, formatOptions) {
 
 function eslintFix(text, eslintConfig) {
   const eslintOptions = {
+    // overrideables
+    useEslintrc: false,
```

Comment:
I think would want to override this so that eslint can use their local `.eslintrc` files. Am i missing something here?